stage1/init/common/path.go

```diff
@@ -54,6 +54,16 @@ func EnvFilePath(root string, appName types.ACName) string {
 	return filepath.Join(common.Stage1RootfsPath(root), RelEnvFilePath(appName))
 }
 
+// RelEnvFilePath returns the path to the environment file for the given app name
```

Comment:
s/RelEnvFilePath/RelEnvFilePathSystemd

that sentence needs to be updated to explain what's different than the previous functions